intents.yaml

```diff
@@ -315,6 +315,7 @@ HassVacuumStart:
       description: "Name of an area"
       required: false
   slot_combinations:
+    none: []
```

Comment:
Are you sure this is required? The `vacuum` domain [is implied](https://github.com/home-assistant/intents/blob/8022db738785c08f4652eea8419b12fac85ed26d/tests/test_language_sentences.py#L104) by default [in the intent definition](https://github.com/home-assistant/intents/blob/8022db738785c08f4652eea8419b12fac85ed26d/intents.yaml#L308).